index.html

```diff
@@ -1426,5 +1426,16 @@ <h2>Security Considerations</h2>
 <div class="issue">TODO: We need to add a complete list of security
 considerations.</div>
     </section>
+    <section class="appendix">
+      <h2>Acknowledgements</h2>
+      <p>
+        Portions of the work on this specification have been funded by the
+        United States Department of Homeland Security's (US DHS) Silicon Valley
+        Innovation Program under contracts 70RSAT20T00000003, and
+        70RSAT20T00000033. The content of this specification does not
```

Comment:
As elsewhere... Keeping all contract numbers in alphanumeric order, with one number per line in the HTML source, will make it easier to see whether a number has been added.
```suggestion
        Innovation Program under contracts 
        70RSAT20T00000003, 
        and
        70RSAT20T00000033. 
        The content of this specification does not
```